### PR TITLE
Remove FIX* variables for fix subfolders

### LINKED ISSUE #2184: Remove FIX* directory variables
### What new functionality do you need?

Replace the use of `FIX*` directory variables with the full path and `FIXgfs`. Example: `FIXam` -> `FIXgfs/am`

Variables to target include:

- `FIXam`
- `FIXaer`
- `FIXgdas`
- `FIXgempak`
- `FIXlut`
- `FIXorog`
- `FIXcpl`
- etc?

### What are the requirements for the new functionality?

Reduced variable settings and continued functionality of system.

- [ ] Updated UFS_UTILS hash with `FIX*` variables removed/updated - https://github.com/ufs-community/UFS_UTILS/pull/899
- [ ] Updated GDASAPP hash with `FIX*` variables removed/updated - https://github.com/NOAA-EMC/GDASApp/pull/932

### Acceptance Criteria

`FIX*` variables are removed and system still works.

### Suggest a solution (optional)

_No response_

### PR DESCRIPTION
**Pull request by KateFriedman-NOAA**
# Description

This PR removes `FIX*` variables for fix subfolders and replaces them with the remaining `FIXgfs` variable and the subfolder name (e.g. `${FIXam}` -> `${FIXgfs}/am`). The UFS_UTILS (https://github.com/ufs-community/UFS_UTILS/pull/899) and GDASApp (https://github.com/NOAA-EMC/GDASApp/pull/932) repos were similarly updated. This PR includes a new UFS_UTILS hash. The updated GDASAPP hash was already committed within the spack-stack/1.6.0 PR #2239.

Resolves #2184

Changes in this PR include:

1. Replace each `FIXtype` with it's respective `FIXgfs/type`; which includes replacing `FIXgempak` with the non-fix path `HOMEgfs/gempak/fix` and replacing `FIXbufrsnd` with `HOMEgfs/fix/product`.
2. Delete `FIXtype` and `FIXgfs` settings and exports, leaving only the initial `FIXgfs` setting in `config.base`.
3. An updated UFS_UTILS hash which includes the following additional updates:

- [Update chgres_cube for fractional grids (](https://github.com/ufs-community/UFS_UTILS/commit/7addff5a639af6ad8c41dc1f36a8ad34967eb9e0)https://github.com/ufs-community/UFS_UTILS/pull/873[)](https://github.com/ufs-community/UFS_UTILS/commit/7addff5a639af6ad8c41dc1f36a8ad34967eb9e0)
- [Version 1.12.0 update (](https://github.com/ufs-community/UFS_UTILS/commit/b15058c925aba78ead87c9bffc47c10d9b52bea8)https://github.com/ufs-community/UFS_UTILS/pull/898[)](https://github.com/ufs-community/UFS_UTILS/commit/b15058c925aba78ead87c9bffc47c10d9b52bea8)

# Type of change

Cleanup

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?

Low-res atmos-only cycled test on both Hera and WCOSS2 with the updated UFS_UTILS and GDASApp hashes.

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [x] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

**Review by CoryMartin-NOAA — APPROVED** [review]
DA related changes look good to me, I approve assuming tests pass, thanks Kate!

**Review by GeorgeGayno-NOAA — APPROVED** [review]
You are pointing to the correct ufs_utils hash. Assuming your tests passed, I approve.

File: ush/wave_prnc_ice.sh [hunk 1]
```diff
@@ -55,7 +55,7 @@ source "$HOMEgfs/ush/preamble.sh"
   echo "Making ice fields."
 
   if [[ -z "${YMDH}" ]] || [[ -z "${cycle}" ]] || \
-     [[ -z "${COM_WAVE_PREP}" ]] || [[ -z "${FIXwave}" ]] || [[ -z "${EXECwave}" ]] || \
+     [[ -z "${COM_WAVE_PREP}" ]] || [[ -z "${FIXgfs}" ]] || [[ -z "${EXECwave}" ]] || \
```
Comment (aerorahul) [inline ush/wave_prnc_ice.sh:58]: should this be `${FIXgfs}/wave`?

Comment (KateFriedman-NOAA) [reply ush/wave_prnc_ice.sh:58]: So it was `${FIXgfs}/wave` but I had this shellcheck warning: https://github.com/NOAA-EMC/global-workflow/security/code-scanning/11613. I removed the `/wave` part to resolve the warning. I think the if-block in question can actually be removed, it seems like it's unnecessary. I will have to touch this block again when I go to remove/replace things like `EXECwave` so I could look at removing those `-z` checks then. Thoughts on that block?

Comment (aerorahul) [reply ush/wave_prnc_ice.sh:58]: handling it later with the `EXECwave` update sounds good to me.

**Review by aerorahul — APPROVED** [review]
looks good.

**Pull request merged**